### DISCUSSION TITLE
Passwords cannot be displayed in the latest Bitwarden Android App when the additional Master Password prompt is enabled

### DISCUSSION THREAD

**Discussion by Krabtus** [dani-garcia/vaultwarden#3915] [Q&A]
**Description**
Since updating to the latest version of Vaultwarden, an issue has been identified that affects the functionality of the latest Bitwarden Android App. This problem occurs when the option for an additional Master Password prompt has been enabled for an entry before passwords can be displayed or copied.

**Steps to Reproduce:**
1. Ensure that Vaultwarden has been updated to the latest version.
2. Open the Bitwarden Android App.
3. Navigate to a saved password entry that you wish to view.
4. If the option for an additional Master Password prompt is enabled within the entry, the problem will become apparent.

**Observed Behavior:**
Once the additional Master Password prompt has been enabled for an entry in Vaultwarden, saved passwords can no longer be displayed or copied. While the Bitwarden App does provide the option to enter the Master Password to copy or view the password, even when the correct Master Password is entered, this does not occur.

**Expected Behavior:**
The Bitwarden Android App should be able to grant access to the saved passwords after the Master Password prompt, as the additional Master Password prompt has been activated. This is crucial for ensuring the security of stored passwords.

This issue impacts usability and security. Thank you in advance for your assistance in resolving this problem.

**Comment by BlackDex** [comment]
See #3904 

f.y.i this is all done client side.
